### PR TITLE
fix: mf-4467 tip button on mirror

### PR DESCRIPTION
**Pull request by UncleBill**
https://mask.atlassian.net/browse/MF-4467

### CODE REVIEW

File: packages/plugins/Tips/src/hooks/useProfilePublicKey.ts [hunk 1]
```diff
@@ -7,10 +7,9 @@ export function useProfilePublicKey(userId?: string) {
     const { networkIdentifier } = useBaseUIRuntime()
     const platform = resolveNetworkToNextIDPlatform(networkIdentifier)
     return useQuery({
-        enabled: !!userId && !!platform,
         queryKey: ['next-id', 'lasted-active', platform, userId],
         queryFn: async () => {
-            if (!userId || !platform) return
+            if (!userId || !platform) return null
```
Comment (guanbinrui) [inline packages/plugins/Tips/src/hooks/useProfilePublicKey.ts]: Why? It seems to cause unnecessary changes.

Comment (UncleBill) [reply packages/plugins/Tips/src/hooks/useProfilePublicKey.ts]: To let react-query cache the result and silence the `undefined` result complaining.

Comment (UncleBill) [reply packages/plugins/Tips/src/hooks/useProfilePublicKey.ts]: This is not absolutely necessary

**Review by guanbinrui — APPROVED** [review]
